src/fidesops/service/connectors/saas_query_config.py

```diff
@@ -166,7 +245,7 @@ def generate_query(
         # create the source of param values to populate the various placeholders
         # in the path, headers, query_params, and body
         param_values: Dict[str, Any] = {}
-        for param_value in current_request.param_values:
```

Comment:
Should you error here too if current_request doesn't exist?

Reply:
Added error check